docs/installation.rst

```diff
@@ -35,15 +71,15 @@ Flask-Session has an increasing number of directly supported storage and client
    * - Storage
      - Client Library
    * - Redis
-     - redis-py_
+     - ``redis`` (via ``redis-py``)
    * - Memcached
-     - pylibmc_, python-memcached_, libmc_ or pymemcache_
+     - ``pymemcache``
```

Comment:
here while we should put this one first and also use it in tests and the default client for installation, we should also list the others as they are still compatible

Reply:
I wonder if the above list on line 44 could be rolled into this table, thereby making it clear what client is being installed and also having a final column for alternative client libraries